index.html

```diff
@@ -184,6 +196,49 @@ <h1>About me</h1>
 				</ul>
 			</div>
 		</div>
+		<footer class="formSection">
+			<div>
+				<h1 class="formHeader">I'm always interested in hearing about new projects, so if you'd like to chat
+					please get in
+					touch.
+				</h1>
+			</div>
+			<form action="https://formspree.io/f/xrgdolbb" method="post">
+				<div class="fullName formBox">
+					<label for="name">
+						<input class="colorbox" type="text" id="name" name="userName" maxlength="30"
+							placeholder="    Full name" required>
+					</label>
+				</div>
+				<div class="email formBox">
+					<label for="formMail">
+						<input class="colorbox" type="email" id="formMail" name="userEmail" placeholder="    Email address"
+							required>
+					</label>
+				</div>
+				<div class="message">
+					<label for="formMessage">
+						<textarea id="formMessage" name="text" maxlength="500"
+							placeholder="Enter text here" required></textarea>
+					</label>
+				</div>
+				<div class="buttonf">
+					<button class="formButton footerTopButton" type="submit">Get in touch</button>
+				</div>
+			</form>
+			<div class="mediaForm">
+				<nav>
+					<ul class="media">
+						<li><a href="#"><img src="./icons/Gitf.png" alt="github"></a></li>
+						<li><a href="#"><img src="./icons/LinkedInf.png" alt="linkedIn"></a></li>
+						<li><a href="#"><img src="./icons/fingerf.png" alt="crossfingers"></a></li>
+						<li><a href="#"><img src="./icons/Twitterf.png" alt="twitter"></a></li>
```

Comment:
Great job 👍 @dgcuenca , Kindly give your form page some padding to make your user interface meet the required [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
**Your design 👇**
![padding](https://user-images.githubusercontent.com/73854999/199350742-5a9fa898-1989-4168-90ea-7fbd864e5a59.jpg)

**Microverse design **
![pad](https://user-images.githubusercontent.com/73854999/199350806-66621736-0a9a-44fe-b02b-791b29e2e7b0.jpg)